esp/services/WsDeploy/WsDeployService.cpp

```diff
@@ -3308,6 +3308,23 @@ bool CWsDeployFileInfo::displaySettings(IEspContext &context, IEspDisplaySetting
           IPropertyTree* pBinding = &iterBindings->query();
           bool flag = false;
 
+          Owned<IPropertyTreeIterator> iterSetting = pBinding->getElements("AuthenticateSetting");
+          ForEach (*iterSetting)
+          {
+            flag = true;
+            break;
+          }
```

Comment:
The above lines (and similar existing lines) would be more concise as:

```
Owned<IPropertyTreeIterator> iterSetting = pBinding->getElements("AuthenticateSetting");
flag = iterSetting->first();
```

Reply:
Agreed, although it can be even more concise (without sacrificing clarity) if 'flag' is removed completely.

Reply:
@jakesmith I've updated the new and existing code with this change.

Reply:
In fact now it's clearer what it's used for, it would be better without the iterator(s) at all.
Better as:

```
    if (!pBinding->hasProp("AuthenticateSetting"))
    {
```

Reply:
@timothyklemm - 1 follow-up comment.

Reply:
@jakesmith Agreed and implemented.

Reply:
@timothyklemm - looks good.